app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  def update_posts_counter
```

Comment:
### [OPTIONAL]
Hint: Just like we have in the post model, after_save :update_likes_counter